### DISCUSSION TITLE
Why seaweed in naming the tool?

### DISCUSSION THREAD

**Discussion by CKHH** [seaweedfs/seaweedfs#6095] [Q&A] [answered]
Why is seaweed used in naming the tool ”SeaweeFS”? Just curious what’s the story behind it. Thanks.

**Comment by CKHH** [comment]
btw it is an awesome tool 👍🏼

**Comment by chrislusf** [comment, marked as answer]
Originally to follow Haystack the name theme, it was named as wild grass (weed). As a Chinese poem says, "Wild fire can not kill them. The spring breeze blows and gives birth again."

Later, Japanese users said weed is just a very bad word. (Not very sure).

So the name was changed to Seaweed, which is "healthier". But the binary name is still "weed".

**Reply by KaKi87** [reply]
> Originally to follow Haystack the name theme

What do you mean ?

**Reply by yalindogusahin** [reply]
Grass spreads across the ancient plain,
Each year it withers, each year it thrives again.
Wildfires cannot burn it all away;
Spring winds return, and it is reborn each day.
Its distant fragrance creeps along old roads,
Bright green reaches ruined city abodes.
Once more I see my friend depart from me—
Thick grass, heavy with farewell’s sorrow, you see.

I didn't know the poem and quite like it! Thanks for sharing!

**Comment by CKHH** [comment]
野火烧不尽，春风吹又生。Interesting, thanks for sharing!

You may be surprise how different chatgpt would answer the same question :)
